proto/ca_service.proto

```diff
@@ -38,16 +45,16 @@ message CertificateSignRequest {
 
 message NodeAgentCredentials {
   oneof node_credential {
-    GcpInstanceCredential gcp_instance_credential = 1;
+    GcpInstanceCredential gcpInstanceCredential = 1;
```

Comment:
The field descriptor naming is inconsistent. So I changed them to be uniform. But after checking protos in mixer repo, looks like the convention is to use snake case. I have updated this and will push a commit.